zetasql/fuzzing/component/fuzz_targets/prepared_expression_target.h

```diff
@@ -24,10 +24,15 @@ namespace zetasql_fuzzer {
 class PreparedExpressionTarget : public FuzzTarget {
  public:
   void Visit(SQLStringArg& sql) override;
+  void Visit(ParameterValueMapArg& arg) override;
   void Execute() override;
 
  private:
   std::unique_ptr<std::string> sql_expression;
+  std::unique_ptr<zetasql::ParameterValueMap> columns =
```

Comment:
auto columns = std::make_unique...

Google-style says you should use auto here, since the type is declared when you use std::make_unique<T>.

Reply:
Nevermind, my mistake.